src/main/webapp/resources/js/apis/samples/samples.ts

```diff
@@ -132,7 +131,13 @@ export const {
  * @param {Object} params
  * @returns {Promise<{}|T>}
  */
-export const fetchMetadataForSample = async ({ sampleId, projectId }) => {
+export const fetchMetadataForSample = async ({
+  sampleId,
```

Comment:
Take a look at how @ericenns is handling `axios` errors.  I think a good example is in `dashboard.ts`, he added better error handling on the server.

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f